### DISCUSSION TITLE
Does the sahi model support the segmentation function of yolov5 7.0?

### DISCUSSION THREAD

**Discussion by ghost** [obss/sahi#838] [General]
May I know if the sahi model supports the segmentation function of yolov5 7.0?
thanks

**Comment by fcakyon** [comment]
Currently not supporting

**Reply by ghost** [reply]
Hi, sir
I would like to ask if there is a plan to add yolo segmentation to the sahi model in the future ? 
Because I have a lot of large images to detect small objects, and I need sahi to detect (segmentation for yolov5)
thanks.

**Reply by fcakyon** [reply]
There is a plan but no ETA.

If anyone is willing for a PR, we can also support him/her 👍

**Reply by ghost** [reply]
Is there anyone willing to help with PR?
I think yolo segmentation is better than detectron2.
thanks.
